### DISCUSSION TITLE
Does this library support large datasets?

### DISCUSSION THREAD

**Discussion by krioru** [patrykandpatrick/vico#501] [Ideas]
Hello. 

I need to display several large static datasets (up to tens of million points each) on the same line chart. Datasets need to be downsampled/decimated to the visible area using algorithms like RDP or LTTB after each zoom or panning event. 

Does this library support something like this? Or maybe there are zoom/panning callbacks, where downsampling could be performed manually?

Thank you.

**Comment by patrickmichalik** [comment]
Hello! Vico is generally intended for more conventional use cases. The drawing logic is optimized for long data sets (only the visible part of a chart is drawn), but I’m afraid dynamic downsampling isn’t supported and isn’t a planned addition for the near future.

**Comment by patrickmichalik** [comment]
As we’re shifting to a more community-driven approach, I’ll be reopening this in the “Ideas” category. To others: Please upvote the discussion if you’d like to see downsampling added.
